data-structures.md

```diff
@@ -319,6 +319,25 @@ A type to represent an amount of filecoin tokens.
 type TokenAmount UInt
 ```
 
+### SectorID
+
+Uniquely identifies a miner's sector.
+
+```sh
+type SectorID uint64
+```
+
+Note: The [`Seal`][seal-link] and [`VerifySeal`][verify-seal-link] proofs
```

Comment:
This feels like a detail that shouldnt be exposed. The proofs code should just accept integers, and whatever weird serialization and padding it needs it should just do internally.